eli5/xgboost.py

```diff
@@ -413,3 +540,13 @@ def _missing_values_set_to_nan(values, missing_value, sparse_missing):
     if not np.isnan(missing_value):
         values[values == missing_value] = np.nan
     return values
+
+
+def shap_prediction_weights(booster, dmatrix, n_targets):
+    feature_weights = booster.predict(dmatrix, pred_contribs=True)
+    if n_targets > 1:
+        score_weights = [i for i in zip(np.sum(feature_weights, axis=2).squeeze(), feature_weights.squeeze())]
+    else:
+        score_weights = [(np.sum(feature_weights), feature_weights.squeeze())]
+
+    return score_weights
```

Comment:
Could you add a test that checks that the score obtained from this function correspond with the prediction obtained from XGBoost? I mean, this is similar to assert that the sum of weights actually corresponds with the prediction that XGBoost returns.

Reply:
@ivanprado  Yes! i rechecked and that is exactly what happens. Do I add a test in `test_xgboost` or just an assert in `xgboost`?

Reply:
Having it in the test would be enough I guess.